docs/book/quick-start.md

```diff
@@ -219,62 +103,49 @@ return [
         ],
     ],
     'controllers' => [
-        // tell the application how to instantiate our controller class
+        // Tell the application how to instantiate our controller class
         'factories' => [
             Application\Controller\HelloController::class => Laminas\ServiceManager\Factory\InvokableFactory::class,
         ],
     ],
 ];
 ```
 
-When the URI path of the request matches `/hello/world`, the specified controller and action will be used. The controller name must be present in the `controllers` list. The associated class will then be instantiated and invoked.
+When the URI path of the request matches `/hello/world`, the specified controller and action will be used.
+The controller name must be present in the `controllers` list.
+The associated class will then be instantiated and invoked.
 
 Learn more about routing [here](https://docs.laminas.dev/laminas-router/routing).
 
-## Tell the Application About our Module
-
-One problem: we haven't told our application about our new module!
-
-By default, modules are not utilized unless we tell the module manager about them.
-As such, we need to notify the application about them.
-
-Remember the `config/application.config.php` file? Let's modify it to add our
-new module. Once done, it should read as follows:
-
-```php
-<?php
-return [
-    'modules' => [
-        'Application',
-        '<module-namespace>',
-    ],
-    // ... other configuration ...
-];
-```
-
-Replace `<module-namespace>` with the namespace of your module.
-
 ## Test it Out!
 
-Now we can test things out! Create a new vhost pointing its document root to the `public` directory
-of your application, and fire it up in a browser. You should see the default homepage template of
-[laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton).
+Create a new vhost pointing its document root to the `public` directory of your application, and fire it up in a browser.
+You should see the default homepage template of [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton).
 
-Now alter the location in your URL to append the path "/hello/world", and load the page. You should
-now get the following content:
+Append the path "/hello/world" to your URL and load the page.
+You should now get the following content:
 
 ```html
 <h1>Greetings!</h1>
 
-<p>You said "foo".</p>
+<p>You said "hello".</p>
 ```
 
-Now alter the location to append "?message=bar" and load the page. You should now get:
+Append "?message=welcome" to your URL.
+You should now get:
 
 ```html
 <h1>Greetings!</h1>
 
-<p>You said "bar".</p>
+<p>You said "welcome".</p>
 ```
 
-Congratulations! You've created your first Laminas MVC module!
+Congratulations!
+You've created your first Laminas MVC controller!
+
+## Learn More
+
+- [Creating custom modules](https://docs.laminas.dev/tutorials/getting-started/modules/)
+- [Controllers](https://docs.laminas.dev/laminas-mvc/controllers)
```

Comment:
If a link points to a page in the same documentation (internal link) then MkDocs should create the URL:
```suggestion
- [Controllers](controllers.md)
```
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages